### PR TITLE
adding info on eat Packages

### PR DESCRIPTION
**Pull request by grewered**

### CODE REVIEW

**Comment by grewered** [comment]
I added Info on all the eat packages to the website. Can you have a look and give Feedback on it?
Is it better to have a single section for each package or is better to have all the info on one page?
is the layout ok?

I still have to translate all the content to English.

**Comment by nickhaf** [comment]
> @nickhaf I added Info on all the eat packages to the website. Can you have a look and give Feedback on it? Is it better to have a single section for each package or is better to have all the info on one page? is the layout ok?

I like the current structure, and also the layout, so I think we can keep it at that for now!

File: _quarto.yml
Comment (nickhaf) [inline _quarto.yml:1]: Maybe we could sort the packages alphabetically?

**Review by nickhaf — CHANGES_REQUESTED** [review]
Looks great, thank you! Just a minor comment, then we can merge.

**Comment by grewered** [comment]
@nickhaf shall I merge then, when the check is through?

**Comment by nickhaf** [comment]
Great, thanks!

**Pull request merged**